docs/windows.md

```diff
@@ -0,0 +1,74 @@
+- Windows has no native support tar files, but does for zip file, so a tar utility and
+  associated filters must be included in the packages directory.
+- Windows has no make utility, so it must be brought in.
+- Windows does not have native support for bash, but does for powershell.
+- Windows does not have native support for utilities like curl, wget, grep, awk, sed and the such, but does for near-equivelent powershell cmdlets which can fullfill the same function. 
+- Windows does not support some os specific python modules, but has support for most others. 
+- There are differences between windows and linux/unix handle semantics and ioctls.
+- There are differences in file system semantics and file system conventions (i.e, "/opt/mesosphere" vs "c:/Mesosphere")
+- Windows does not support systemd or journald.
+- Windows docker operation is very different from linux
+- Many of the packages used in the linux deployment are different in the windows environment.
+- Windows machines usually prefer pre-built packages rather than in-situ builds from source.
+
+## Design Goals
+
+The design goals of goals of the Windows deployment are:
+- Make differences as consistent and predictable as possible. In principle a linux DC/OS system administrator should be able to learn a few transformation rules between the two environments and predict how to install, modify and operate a Windows host using the same skills they have already developed in linux.
+- Make the deployment also Windows native. A Windows sysadmin should be able to apply their knowlege hand habits as much as possible.
+- Make the deployment functionally equivalent (so far as possible) to the linux deployment.  A DC/OS UI user should not be able to obviously tell the difference between a linux and windows host without looking at the host attributes.
+
+The installation adds to those goals:
+- The deployment should be feasible on a Windows host without additional preparation. As a result, the deployment must use only tools available on a freshly installed windows machine.
+
+## changes to the dcos/dcos repository
+
+The package tree, top level scripts, and configuration files are, as much as possible, organised in a parallel manner to the
+linux build process, but they are not the same, so shell scripts are shadowed by powershell scripts of the same name, ie build_local becomes build_local.ps1, prep_local becomes prep_local.ps1 and so on.  
+
+- The package tree is paralleled by packages.windows. 
+  - In each directory we have a buildinfo.json parallel to the linux buildinfo.json, but usually different as windows requires different dependencies, packages, git repositories or branches, and commit shas from the linux equivelant. 
+  - The build script located in each of the pacakge directories is paralleled by a build.ps1 script which is functionally similar.
+  - In the packages.windows tree there are setupinfo.json and setup.ps1 scripts wwhich provide setup information at package 
```

Comment:
Looks like the package `setupinfo.json` and `setup.ps1` files are gone. Should this line be removed?

Reply:
yup!